.github/workflows/code-lint.yml

```diff
@@ -28,3 +28,20 @@ jobs:
         . ./ci/common.sh
         export_or_prefix
         make lint
+
+  sc-lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.4.0
+        with:
+          submodules: true
+
+      - name: Shellcheck code
+        run: |
+          scversion="latest"
+          wget -O- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.x86_64.tar.xz" | tar -xJv
+          cp -av "shellcheck-${scversion}/shellcheck" /usr/local/bin/
```

Comment:
can we use `apt` to install `spellcheck` ?
like
```bash
apt install -y shellcheck
```

Reply:
This way is ok to install `shellcheck`, but I'm not sure its repository updates to latest version.

1. latest version may add new check rules and it covers the before.
2. different developers use different version, the latest covers them.

The package must uniformly install from repo unless no supported, I'll improve installation.

Reply:
LGTM, let keep it this way. 

Thanks for replying.